### PR TITLE
Changed alignment of search button

### PR DESCRIPTION
**Pull request by Sarvesh-Ghildiyal**
Targets #{2282}

### Description
Alligned search button with search placeholder.
### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comments.

File: resources/views/knowledgecafe/library/books/index.blade.php [hunk 1]
```diff
@@ -18,11 +18,11 @@
         @endcan
     </div>
     <div class="row mt-3 mb-2 px-2">
-        <div class="col-lg-4 col-md-5 col-sm-6 col-xs-12 mr-2 mb-2 p-2 d-flex justify-content-center align-items-center">
+        <div class="col-lg-4 col-md-5 col-sm-6 col-xs-12 mr-2 mb-2 p-2  d-flex justify-content-center align-items-center">
```
Comment (gautam071) [inline resources/views/knowledgecafe/library/books/index.blade.php:21]:
```suggestion
        <div class="col-lg-4 col-md-5 col-sm-6 col-xs-12 mr-2 mb-2 p-2 d-flex justify-content-center align-items-center">
```

**Review by gautam071 — CHANGES_REQUESTED** [review]
remove this unwanted space

File: resources/views/knowledgecafe/library/books/index.blade.php (hunk 1 above)
Comment (Sarvesh-Ghildiyal) [reply resources/views/knowledgecafe/library/books/index.blade.php]: Okay! i have made the changes, but can you please explain how exactly does it affect the code?

**Review by Sarvesh-Ghildiyal — COMMENTED** [review]
Done

**Pull request closed without merging**